package.json

```diff
@@ -6,19 +6,17 @@
   "homepage": "http://metamask.github.io/Add-Token",
   "dependencies": {
     "@material-ui/core": "^3.0.0",
+    "@metamask/detect-provider": "^1.2.0",
     "etherscan-link": "^1.0.3",
     "ethjs-query": "^0.3.8",
-    "prop-types": "^15.6.2",
```

Comment:
Nit: It might be nice to keep this around and document the prop types. Not especially important though.

Reply:
Sorry I wasn't clear :sweat_smile: I meant it might be useful to keep at around and actually define `propTypes` for each component. It looks like it's still not used at the moment, and there's not much point in keeping it if it isn't used.